packages/notifications/package.json

```diff
@@ -33,11 +33,11 @@
         "prop-types": "^15.6.2",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
-        "redux": ">=4.2.0",
-        "react-redux": ">=7.2.9"
+        "redux": ">=5.0.1",
+        "react-redux": ">=9.1.0"
```

Comment:
This could introduce conflicts with NPM and local versions of redux. I suggest loosening the version range even more, but the version range. `"^4.2.0 || ^5.0.0"` etc. Alternatively, we could use the "*" range but that would be probably way too much. This is specific to the notifications package as it is installed by almost every HCC service.

We will remove redux dependencies from the notifications package at some point. No free cycles any time soon though.